gilded-rose-llc-frontend/__tests__/index.tests.tsx

```diff
@@ -1,10 +1,48 @@
-import { render, screen } from "@testing-library/react";
-import Home from "../pages/index";
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Home from '../pages/index'
 
-describe("Home", () => {
-  it("should pass this test meaning that the testing framework is working", () => {
-    render(<Home />);
+describe('Home', () => {
+  it('should render a heading of Gilded Rose', () => {
+    render(<Home items={[]} />)
 
-    screen.debug();
-  });
-});
+    const heading = screen.getByRole('heading', { name: /Gilded Rose/i })
+    expect(heading).toBeInTheDocument()
+  })
+
+  it('should render all of the items on the page', () => {
```

Comment:
Question: Do we need this test? In my opinion, the test below already implied all items will be rendered.

Reply:
My goal with this was to test that every item was rendered on the page so I should have included more than 2 items passed but in reality, this test will limit our flexibility with future changes since this test will break if we end up adding pagination in the future.

This test was removed in [0c7a0c1](https://github.com/jramnani/GildedRoseJavaLLC/pull/32/commits/0c7a0c18e8b87d8e05502f5bf4652cf9ef0f44b1)

Reply:
👍